zephyr/rz/rz_cfg/fsp_cfg/bsp/rzg3s/bsp_clock_cfg.h

```diff
@@ -4,55 +4,57 @@
-#define BSP_CFG_DIVSDHI1_SET_DIV (BSP_CLOCKS_SDHI1_DIV_2) /* SD1CLK Div /2 */
-#define BSP_CFG_CLOCK_SD1CLK_HZ (133333333) /* SD1CLK 133333333Hz */
-#define BSP_CFG_SEL_SDHI2_SET_SOURCE (BSP_CLOCKS_SOURCE_CLOCK_PLL2_266) /* SD2CLK Sel: 266MHz */
-#define BSP_CFG_DIVSDHI2_SET_DIV (BSP_CLOCKS_SDHI2_DIV_2) /* SD2CLK Div /2 */
-#define BSP_CFG_CLOCK_SD2CLK_HZ (133333333) /* SD2CLK 133333333Hz */
-#define BSP_CFG_CLOCK_PLL3_HZ (1600000000) /* PLL3 1600000000Hz */
-#define BSP_CFG_CLOCK_M0CLK_HZ (200000000) /* M0CLK 200000000Hz */
-#define BSP_CFG_DIVPL3A_SET_DIV (BSP_CLOCKS_PL3A_DIV_1) /* P1CLK Div /1 */
-#define BSP_CFG_CLOCK_P1CLK_HZ (200000000) /* P1CLK 200000000Hz */
-#define BSP_CFG_DIVPL3B_SET_DIV (BSP_CLOCKS_PL3B_DIV_1) /* P2CLK Div /1 */
-#define BSP_CFG_CLOCK_P2CLK_HZ (100000000) /* P2CLK 100000000Hz */
-#define BSP_CFG_DIVPL3C_SET_DIV (BSP_CLOCKS_PL3C_DIV_1) /* P3CLK Div /1 */
-#define BSP_CFG_CLOCK_P3CLK_HZ (200000000) /* P3CLK 200000000Hz */
-#define BSP_CFG_CLOCK_ATCLK_HZ (400000000) /* ATCLK 400000000Hz */
-#define BSP_CFG_CLOCK_ZTCLK_HZ (100000000) /* ZTCLK 100000000Hz */
-#define BSP_CFG_SELOCTA_SET_SOURCE (BSP_CLOCKS_SOURCE_CLOCK_PLL3_266) /* OCCLK Sel: 266MHz */
-#define BSP_CFG_DIVOCTA_SET_DIV (BSP_CLOCKS_OCTA_DIV_8) /* OCCLK Div /8 */
-#define BSP_CFG_CLOCK_OC0CLK_HZ (33333333) /* OC0CLK 33333333Hz */
-#define BSP_CFG_CLOCK_OC1CLK_HZ (16666666) /* OC1CLK 16666666Hz */
-#define BSP_CFG_SELSPI_SET_SOURCE (BSP_CLOCKS_SOURCE_CLOCK_PLL3_400) /* SPICLK Sel: 400MHz */
-#define BSP_CFG_DIVSPI_SET_DIV (BSP_CLOCKS_SPI_DIV_4) /* SPICLK Div /4 */
-#define BSP_CFG_CLOCK_SPI0CLK_HZ (100000000) /* SPI0CLK 100000000Hz */
-#define BSP_CFG_CLOCK_SPI1CLK_HZ (50000000) /* SPI1CLK 50000000Hz */
-#define BSP_CFG_CLOCK_PLL4_HZ (800000000) /* PLL4 800MHz */
-#define BSP_CFG_SELPL4_SET_SOURCE (BSP_CLOCKS_SOURCE_CLOCK_PLL4) /* Sel: PLL4 */
-#define BSP_CFG_CLOCK_S0CLK_HZ (400000000) /* S0CLK 400000000Hz */
-#define BSP_CFG_CLOCK_PLL6_HZ (500000000) /* PLL6 500000000Hz */
-#define BSP_CFG_DIVPL6A_SET_DIV (BSP_CLOCKS_PL6A_DIV_1) /* I2CLK Div /1 */
-#define BSP_CFG_CLOCK_I2CLK_HZ (250000000) /* I2CLK 250000000Hz */
-#define BSP_CFG_DIVPL6B_SET_DIV (BSP_CLOCKS_PL6B_DIV_1) /* I3CLK Div /1 */
-#define BSP_CFG_CLOCK_I3CLK_HZ (250000000) /* I3CLK 250000000Hz */
-#define BSP_CFG_CLOCK_HPCLK_HZ (250000000) /* HPCLK 250000000Hz */
-#define BSP_CFG_CLOCK_OSCCLK2_HZ (8000000) /* OSC2 8000000Hz */
+ #define BSP_CLOCK_CFG_H_
+ #define BSP_CFG_CLOCKS_SECURE           (0)                                                          /* Secure clock config */
+ #define BSP_CFG_CLOCKS_OVERRIDE         (0)                                                          /* Override clock config */
+ #define BSP_CFG_CLOCK_OSCCLK_HZ         DT_PROP_OR(DT_NODELABEL(oscclk), clock_frequency, 24000000)  /* OSC 24000000Hz */
+ #define BSP_CFG_CLOCK_PLL1_HZ           (1100000000)                                                 /* PLL1 1100000000Hz */
```

Comment:
@nhutnguyenkc : I would like to confirm that OSC and PLL1 are fixed values as in HWM but the method to get value is different. OSC get from DT node, PLL1 is fixed definition. Is there any special reason here?

Reply:
@thaoluonguw , there is no special reason for it. I just thought that the BSP_CFG_CLOCK_PLL1_HZ is presenting for the frequency after PLL1 block and is the result of the calculation of OSC and PLL1. For example,

`#define BSP_CFG_CLOCK_PLL1_HZ     (BSP_CFG_CLOCK_OSCCLK_HZ/2)`

But in case of RZ/G3S, no formula is provided, and only fixed value is showed in HWUM. Therefore, I would like to set the fixed value for it.

Please let me know if you have any suggestion.